### DISCUSSION TITLE
Garagentore als Objekte?

### DISCUSSION THREAD

**Discussion by ChrisKoh83** [LeonardSchwenk/public-gruppenadressen-generator#34] [Q&A]
Wie würdet ihr Garagentore einbinden?

Ich sehe bisher noch kein Passendes Objekt dazu. Oder verstehen ich das System noch nicht?!

**Comment by sebastiantheobald** [comment]
Hi @ChrisKoh83 , danke für deine Nachricht! 
Bisher ist das noch nicht vorgesehen das stimmt. Können wir gerne ins nächste Update mit aufnehmen. Was würdest du denn benötigen? Ich schlage vor, dass unter "Verbraucher" einzuordnen?

**Reply by ChrisKoh83** [reply]
Hi @sebastiantheobald,

ich bin noch ziemlich neu was das Thema KNX angeht...

Wären Garagentore nicht etwas was man zu "Sicherheit" zuordnen würde? 
Weil Ein Verbraucher ist es ja nicht wirklich, oder?
Eine Haustür mit Motorschloss könnte man da auch einbinden.

Weiter würde mich noch spontan einfallen:

- Kritische Lasten: (Also zB Verbraucher die man über Notstrom versorgt: Kühlschrank/Zugangssysteme/EDV/...)
- Beleuchtung: (Ambientebeleuchtung)
- Kontakte: (VOC-Sensor/CO2)

Grüße
Christian

**Reply by ChrisKoh83** [reply]
Was auch noch cool wäre bei Verbrauchern auch (per Schiebeschalter) eine Verbrauchsmessung mit auszuwählen.
